Dockerfile

```
@@ -0,0 +1,22 @@
+
+ARG UBUNTU_VER=20.04
+FROM ubuntu:$UBUNTU_VER
+
+ARG PYTHON_VER=3.9
+ARG SOURCE=https://download.pytorch.org/whl/cpu
+ENV DEBIAN_FRONTEND=noninteractive
+
+# hadolint ignore=DL3008
+RUN apt-get update && \
+    apt-get install curl python$PYTHON_VER python$PYTHON_VER-dev python$PYTHON_VER-distutils g++ ffmpeg libsm6 libxext6 libgl1-mesa-glx -y && \
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+    python$PYTHON_VER get-pip.py && \
+    rm get-pip.py
```

```suggestion
    rm get-pip.py && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```
changes above will remedy the issue below
```bash
Dockerfile:10 DL3009 info: Delete the apt-get lists after installing something
```